tests/rc_test/.in_totorc

```diff
@@ -1,5 +1,4 @@
 [settings]
 artifact_base_path=r/c/file
 ARTIFACT_EXCLUDE_PATTERNS=r:c:file
-LINK_CMD_EXEC_TIMEOUT=20
```

Comment:
@adityasaky, looks like we forgot to remove this in f88138c90861953c77a1384ea2fcc58126e6fe59.

Reply:
@chasen-bettinger, you can revert this change and we'll remove the fixture in a separate PR.

Reply:
whoops, yeah